src/YoutubeAtom.tsx

```diff
@@ -164,9 +165,35 @@ export const YoutubeAtom = ({
     const [hasUserLaunchedPlay, setHasUserLaunchedPlay] = useState<boolean>(
         false,
     );
+    const [hasUserHovered, setHasUserHovered] = useState<boolean>(false);
     const player = useRef<YoutubePlayerType>();
 
+    const hasOverlay = overrideImage || posterImage;
+
+    // Show the overlay if:
+    // 1) We have one
+    // AND
+    // 2) it hasn't been clicked upon
+    const showOverlay = hasOverlay && !hasUserLaunchedPlay;
+    // Show a placeholder if:
+    // 1) We don't have an iframe source yet (probably because we don't have consent)
+    // AND
+    // 2) There's no overlay to replace it with or the reader clicked to play but we're
+    //    still waiting on consent
+    const showPlaceholder = !iframeSrc && (!hasOverlay || hasUserLaunchedPlay);
+    // Load the you tube iframe if:
+    // 1) We have a source string defined (Eg. We have consent)
```

Comment:
Might be quite pedantic, but `in extensio` might be more accurate than `exempli gratia`. 🤷 

```suggestion
    // 1) We have a source string defined (i.e. we have consent)
```